hello/CMakeLists.txt

```
@@ -1,7 +1,7 @@
 # CMakeのバージョンを設定
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.10.2)
```

aaaaaaaaaaaaaaaaaaaaaa
```suggestion
cmake_minimum_required(VERSION 3.10.2)
```